public/catch-js-errors.js

```diff
@@ -1,6 +1,7 @@
 ( function() {
 	var savedWindowOnError = window.onerror,
 		assignment = 1,
+		localStorageAssignment = 'false',
```

Comment:
Maybe `errorLogger`? The fact that we stored this in localStorage isn't super important.